lib/cli.rb

```diff
@@ -530,13 +530,9 @@ def self.check_port_validity(port)
     c.action do |_global_options, options, args|
       name = shift_arg(args, "NAME")
       description = SystemDescription.load(name, system_description_store)
-      exporter = StaticHtml.new(description)
-
-      task = ExportTask.new(exporter)
```

Comment:
ExportTask contains some unrelated logic, like checking for scopes, adding a sub directory below the target directory and applying filters.